dist/index.js

```diff
@@ -4606,11 +4606,17 @@ function run() {
             // ament_cmake should handle this automatically, but we are seeing cases
             // where this does not happen. See issue #26 for relevant CI logs.
             core.addPath(path.join(rosWorkspaceDir, "install", "bin"));
-            const colconBuildCmd = `colcon build --event-handlers console_cohesion+ --symlink-install --packages-up-to ${packageNameList.join(" ")} ${extra_options.join(" ")} --cmake-args ${extraCmakeArgs}`;
+            let colconBuildCmd = `colcon build --event-handlers console_cohesion+ --symlink-install \
```

Comment:
Optional - now would be a good time to extract building those commands into functions, and adding a unit test for those.

Reply:
That's the idea. I'm doing some cleanup before hand tho. Will open some issues to track.